delete_book.php

```diff
@@ -0,0 +1,15 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The SQL query construction is vulnerable to SQL injection due to direct inclusion of user input.

Recommendations:
- **Use Prepared Statements**: Modify the `DELETE` query to use prepared statements to prevent SQL injection. This ensures safer database interactions.